shell/app/theme-color.mjs

```diff
@@ -23,16 +23,16 @@ const colors = {
   red: '#d84b65',
   'red-1': 'rgba(216,75,101,0.1)',
   danger: 'rgba(216,75,101,1)',
-  error: 'rgba(216,75,101,1)',
+  error: '#d84b65',
```

Comment:
Should keep rgba type, you can add `d84b65 ` in comment.

Reply:
these colors are used in some places as arguments To the function colorToRgb(color: string, opacity?: number), and can only be of hexadecimal type